ximage/labels/labels.py

```diff
@@ -129,6 +130,7 @@ def _get_label_value_stats(arr, label_arr, label_indices=None, stats="area"):
             index=label_indices,
             func=stats,
             out_dtype=float,
+            default=None,
```

Comment:
Maybe we could regroup `out_dtype`, `default `and `pass_positions` inside a `labeled_comprehensions_kwargs `dictionary ... here use `**labeled_comprehensions_kwargs` (which if `{}` takes the current defaults) and add the argument `labeled_comprehensions_kwargs={}` to `_get_labels_stats`, `_get_labels`, `_xr_get_labels`, `label` (and the accessor)

Reply:
Done